test/Microsoft.AspNet.Razor.Test/Generator/CSharpTagHelperRenderingTest.cs

```diff
@@ -552,11 +561,22 @@ public void TagHelpers_WithinHelpersAndSections_GeneratesExpectedOutput()
             RunTagHelperTest("TagHelpersInSection", tagHelperDescriptors: tagHelperDescriptors);
         }
 
-        private static IEnumerable<TagHelperDescriptor> BuildPAndInputTagHelperDescriptors(string prefix)
+        private static IEnumerable<TagHelperDescriptor> BuildPAndInputTagHelperDescriptors(
+            string prefix,
+            bool switchAttributes)
```

Comment:
Test method, just default this to false and don't specify it for the existing `IEnumerable<TagHelperDescriptor>` properties.